packages/terra-consumer-nav/src/Nav.jsx

```diff
@@ -93,6 +94,7 @@ class Nav extends React.Component {
             title: '',
             content: <div />,
           },
+        // eslint-disable-next-line react/no-unused-state
         numberOfLinks,
```

Comment:
This one was interesting.. not sure if this is _actually used_ or not? Opted to leave it in, but disable lint

Reply:
hmm, should be removed